galsim/wcs.py

```diff
@@ -2321,3 +2321,14 @@ def __getstate__(self):
     def __setstate__(self, d):
         self.__dict__ = d
         self._initialize_funcs()
+
+def compatible_wcs(wcs1, wcs2):
```

Comment:
I think `galsim.wcs.compatible_wcs` is slightly redundant with two `wcs` parts.  Perhaps if it lives in `galsim.wcs` (which seems appropriate), then it doesn't need the `_wcs` suffix.